src/Automation/Automation/Model/UpdateManagement/SoftwareUpdateRun.cs

```diff
@@ -57,6 +57,6 @@ internal SoftwareUpdateRun(string resourceGroupName, string automationAccountNam
 
         public SoftwareUpdateRunStatus Status { get; set; }
 
-        public SoftareUpdateConfigurationRunTasks Tasks { get; set; }
+        public SoftwareUpdateConfigurationRunTasks Tasks { get; set; }
```

Comment:
why we need change SoftareUpdateConfigurationRunTasks to SoftwareUpdateConfigurationRunTasks, this change will introduce breaking changes.

Reply:
If this change is necessary
- Please suppress the detection of breaking change in azure-powershell/tools/StaticAnalysis/Exceptions/Az.Automation/BreakingChangeIssues.csv. 
- Please wait for our major release (about middle of May), because breaking changes are only allowed on major release.

Reply:
Yes, this is necessary as a spelling correction was made in the swagger. Have added the BreakingChangeIssues.csv file.

Reply:
We cannot measure the usage of `SoftareUpdateConfigurationRunTasks` but we assume the impact is not small because that parameter was added 2 years ago. `Get-AzAutomationSoftwareUpdateRun` has been called 240K times in the past 30 days and `Get-AzAutomationSoftwareUpdateMachineRun` has been called 73K times. Given the impact and importancy of Az.Autoamtion, we recommend that you add this feature in the mid of May.

Reply:
Created 'SoftareUpdateConfigurationRunTasks' based on 'SoftwareUpdateConfigurationRunTasks' to avoid the breaking change temporarily